test/test_jit.py

```diff
@@ -40,6 +40,7 @@
 from jit.test_isinstance import TestIsinstance  # noqa: F401
 from jit.test_cuda import TestCUDA  # noqa: F401
 from jit.test_hash import TestHash  # noqa: F401
+from jit.test_complex import TestComplex  # noqa: F401
```

Comment:
Can you undo the formatting changes? You can add parts of files to your commit using `git add <files> -p`